tests/pkg_ucglib/logo.h

```diff
@@ -0,0 +1,1178 @@
+/*
```

Comment:
any reason this is a header (instead of .c)?

Reply:
Not really. Most converters I tried are called something like `img2header` so I went for a header file.

Is there a difference?

Reply:
Ping @kaspar030 :-)

Reply:
no difference.